### PR TITLE
Add file search functionality by adding a separate collection for files

### PR DESCRIPTION
**Pull request by kartik-gupta-ij**

### CODE REVIEW

File: code_search/index/file_uploader.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   
+    qdrant_client = QdrantClient(
+        QDRANT_URL,
+        api_key=QDRANT_API_KEY,
+    )
+
+    collection_name = QDRANT_FILE_COLLECTION_NAME
+    input_file = Path(DATA_DIR) / "rs_files.json"
+
+    input_file = Path(DATA_DIR) / input_file
```
Comment (generall) [inline code_search/index/file_uploader.py]: duplication?

File: tools/index_qdrant.sh [hunk 2]
```diff
@@ -11,6 +11,10 @@ SCRIPT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 ROOT_PATH=$SCRIPT_PATH/..
 
 
+python -m code_search.index.files_to_json
```
Comment (generall) [inline tools/index_qdrant.sh:14]: where the original files are coming from?

File: code_search/index/file_uploader.py (hunk 1 above)
Comment (kartik-gupta-ij) [reply code_search/index/file_uploader.py]: Fixed!!

File: tools/index_qdrant.sh (hunk 2 above)
Comment (kartik-gupta-ij) [reply tools/index_qdrant.sh:14]: https://github.com/qdrant/demo-code-search/blob/f93eab28ecdff42f63c8e7b0dde616cc8fc5aeaf/code_search/index/files_to_json.py#L28

**Pull request merged**